Containerfile

```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
 FROM scratch
 COPY --from=lightspeed-rag-builder /workdir/vector_db/ocp_product_docs /rag/vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-builder /workdir/embeddings_model /rag/embeddings_model
+
+# Labels for enterprise contract
+LABEL com.redhat.component=openshift-lightspeed
```

Comment:
Is it ok to have the same component as the operator?

In service we have:
com.redhat.component=openshift-lightspeed-service